### PR TITLE
Adds footer and contact form to device (mobile and tablet) version

### PR DESCRIPTION
**Pull request by GerardoGomezG**
# Footer mobile version

imagen

# Adds general changes for tablet version display

imagen

### CODE REVIEW

**Comment by GerardoGomezG** [comment]
## Changes were done as requested @danyhoshi 
## Description is shown on each commit message.

# Tablet Version
Screenshot 2023-04-03 at 12-22-15 Gerardo's Portfolio

# Mobile Version
Screenshot 2023-04-03 at 12-25-10 Gerardo's Portfolio Cellphone

**Review by danyhoshi — APPROVED** [review]
# Look soo good @GerardoGomezG  👍

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "hint": "^7.1.3",
+    "hint": "^7.1.8",
```
[the remark this answers is not in the document]
Comment (GerardoGomezG) [reply package.json:13]: @danyhoshi Once I pimped my PC with new hardware and installed everything from scratch, a message appeared in VSC bash terminal suggesting to perform an update through a command line. So I updated it and it seems to be the reason of that change in hint version.

File: styles.css [hunk 2]
```diff
@@ -178,20 +181,317 @@ hr {
...
+  color: #f0e3ca;
+  font-size: 1rem;
+  resize: none;
+}
+
+@media screen and (min-width: 501px) {
+  header {
+    display: grid;
+    grid-template-columns: auto;
+    width: 100vw;
```
[the remark this answers is not in the document]
Comment (GerardoGomezG) [reply styles.css:245]: Recommendation applied @danyhoshi !

File: styles.css [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
+  height: 2.5em;
+}
+
+header {
+  position: sticky;
+  margin: 0;
+  padding-top: 1em;
   top: 0;
   left: 0;
-  justify-content: space-between;
```
[the remark this answers is not in the document]
Comment (danyhoshi) [reply styles.css:15]: Fine!!! It's working now! 👍

**Review by danyhoshi — APPROVED** [review]
# Good job @GerardoGomezG !!!
I token looking again and was resolved
thumb up

**Pull request merged**